lua/astrocommunity/pack/ansible/init.lua

```diff
@@ -0,0 +1,24 @@
+local utils = require "astronvim.utils"
+
+return {
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if opts.ensure_installed ~= "all" then
+        opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, "yaml")
+      end
+    end,
+  },
+  {
+    "jay-babu/mason-null-ls.nvim",
+    opts = function(_, opts) opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, "ansiblelint") end,
+  },
+  {
+    "williamboman/mason-lspconfig.nvim",
+    opts = function(_, opts) opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, "ansiblels") end,
+  },
+  {
+    "pearofducks/ansible-vim",
+    ft = "yaml.ansible",
```

Comment:
```suggestion
    "pearofducks/ansible-vim",
    init = function()
      -- add vim.filetype
    end,
    ft = "yaml.ansible",
```

I think we can put the filetype add to this plugins initialization to get the lazy loading

Reply:
Oh also could you add ansible-vim mention to the readme